packages/fxa-settings/.storybook/design-guide/pages/Introduction.tsx

```diff
@@ -82,4 +82,5 @@ export const Introduction = () => (
   </Page>
 );
 
-export default () => Introduction();
+export const introduction = () => Introduction();
```

Comment:
Q: should this be exported too?
Doesn't seem consistent with above patterns and i dont think it was exported previously as a named export.

Reply:
The linter error was essentially that the default export had no name.  IMO it's fine to export by name as well as being exported as the default.

Reply:
Ah, apologies. I didn't realize that this was just a storybook file. :yolo:

Reply:
VS Code does not warn about the lint error.  I presume it's the same for the author of that line.  Should we add fxa-setting linting to the pre-commit hook then?

Reply:
This has been added to the husky config for lint-staged.